bpo-29474: Improve documentation for weakref.WeakValueDictionary

Mariatta opened the pull request:
There were some grammatical errors in weakref.WeakValueDictionary
documentation.

This is to be applied to 2.7

dianaclarke commented:
@Li4ick: This is a read-only mirror of cpython. Please see the devguide (https://docs.python.org/devguide/index.html) for more information about contributing.

Mariatta committed:
bpo-29474: Improve documentation for weakref.WeakValueDictionary

There were some grammatical errors in weakref.WeakValueDictionary
documentation.

This is to be applied to 2.7

zware reviewed (approved):
Trusting that the sentence makes sense in context, this does indeed fix the grammar.  LGTM.

Mariatta commented:
Thanks for the reviews, @zware :)
